### ISSUE TITLE
Feature Enhancement: Provide an option to open links in the foreground by Holding down "CTRL", if the option open new tab in the background is selected.

### ISSUE THREAD

**Issue by Droyk** [AdeelH/find-on-reddit#38]
![image](https://user-images.githubusercontent.com/15848350/154783010-e97fb956-a9f6-4179-bf1d-09f59702f4ba.png)

usually, I like to open things in the background but sometimes I want to jump to it straight so I thought wouldn't it be great if a user can just hold ctrl & open in background mode changes to open in a new tab!

**Comment by AdeelH** [comment]
This is not standard or intuitive behavior for `ctrl + click`. Besides, I don't think it takes too much energy to switch to a different tab, especially if you are only doing it sometimes.

**Comment by Droyk** [comment]
I agree with it's not the standard but that shouldn't mean it shouldn't get implemented. I mean if one can hold ctrl to open the link in the background tab in the normal mode then the reverse should also sense. it shouldn't be in the realm of impossibility. holding down ctrl should open the link in the foreground rather than in the background when opening the link in the new tab option & in background mode option is checked.

& saying it's not the standard is an irony in itself! this extension is a niche how many users are using it!
![image](https://user-images.githubusercontent.com/15848350/154854767-0754786c-53f5-42d5-bf99-d6ee6ab4e182.png)

Your own extension isn't a standard! standard solves things slowly that's why extension stores exist to make it better than what the standard provides. let's take an example of [power toys ](https://github.com/microsoft/PowerToys) the power toys don't come with Windows pre-installed. which means only a very small amount of users are using it! to escape from that standard to make their lives productive overall!

there are so many features in it the above tool that won't be added in the windows by default but they are hugely productive & aren't a standard.

coming down to the energy bit well the option would just get things past faster. it's easier that way. I mean I hardly need to do anything from my part just pressing & holding down the ctrl would bring the page whereas first I need to click then & then I need to see where is the tab in the long list of tabs and then I hit that tab to make it active. the former option is faster to actuate.  it's just a small quality-of-life improvement. & when I say sometimes it means at least 10 times in the period of 18 hours whereas 100+ times opening the link in the background tab in the same period.

Providing an option to enable it or not would make things easier it's a +1 for UX!
![image](https://user-images.githubusercontent.com/15848350/154860708-91619a64-2cbc-4464-a2e2-8c982527f8f2.png)
